lib/archive.rb

```diff
@@ -10,8 +10,12 @@
 #
 module Archive
   def self.get_files(archive_path)
-    archive_type = get_archive_type(archive_path)
-    archive_extract = get_archive(archive_path, archive_type)
+    begin
+      archive_type = get_archive_type(archive_path)
+      archive_extract = get_archive(archive_path, archive_type)
+    rescue Zip::Error => e
```

Comment:
Should we rescue other errors here as well? i.e. `Gem::Package::TarInvalidError` or others that my be returned

Reply:
the tar error is thrown and caught later in the file, zips die here